api/openapi.generated.yml

```diff
@@ -65,12 +65,6 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
           description: Validation error
-        '401':
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/ErrorResponse'
-          description: Authentication error
```

Comment:
Ah - forgot about one other thing we have to do with the multiauth to make it build the swagger docs happily.

Could you add this to the routes you changed to multiauth?:
```py
@whatever_blueprint.doc(security=api_key_multi_auth_security_schemes)
```
The `@whatever_blueprint.auth_required(whatever)` adds this normally, but that doesn't work with multiauth, so we have to manually add the docs ourselves.